src/feedback.js

```diff
@@ -152,7 +153,7 @@ export default class Feedback {
 						<p>${ feedbackType.icon } ${ feedbackType.text }</p>
 					</div>
 					<div class="feedback-content">
-							${ this.options.emailField ? '<input id="feedback-email" type="email" name="email" placeholder="Email address (optional)">' : '' }
+							${ this.options.emailField ? '<input id="feedback-email" type="email" name="email" placeholder="${ this.options.emailplaceholder }">' : '' }
```

Comment:
This doesn't quite work yet, but almost.

Since we are using a JS variable in the string, you need to replace the `'` with `. They are [template literals](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Template_literals).

So it should be:

```
${ this.options.emailField ? `<input id="feedback-email" type="email" name="email" placeholder="${ this.options.emailplaceholder }">` : '' }
```